dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -17,8 +17,8 @@
  * A node for new object creation.
  *
  * <pre>
- *   <em>new constructor(arg1, arg2, ...)</em>
- *   <em>enclosingExpression.new constructor(arg1, arg2, ...)</em>
+ *   <em>new identifier(arg1, arg2, ...)</em>
```

Comment:
What about a generic type and a generic constructor? How are the class and method type arguments represented?

Reply:
https://github.com/eisop/jdk/blob/df7ba5244ce1bffb38e60e8d96c15e2c972d2b64/src/jdk.compiler/share/classes/com/sun/source/tree/NewClassTree.java#L62
has a `getTypeArguments` method, which I think are the constructor type arguments.
The class type arguments are contained within `identifier`, right?
What concrete subclasses of `Node` can `identifier` refer to? `ParameterizedTypeNode` and what else?
Is `ClassNameNode` used if it's a non-generic class?
Can you please investigate with a few example and improve the documentation of this class?

Reply:
Thanks!

Reply:
Done in my local.